pyzx/graph/diff.py

```diff
@@ -80,13 +84,13 @@ def calculate_diff(self, g1: BaseGraph[VT,ET], g2: BaseGraph[VT,ET]) -> None:
 				pos2 = g2.qubit(v), g2.row(v)
 				self.changed_pos[v] = pos2
 
-		for e in new_edges:
-			if e in old_edges:
-				if g1.edge_type(e) != g2.edge_type(e):
-					self.changed_edge_types[e] = g2.edge_type(e)
-			else:
-				if g2.edge_type(e) != EdgeType.HADAMARD: # We take Hadamard edges to be the default
-					self.changed_edge_types[e] = g2.edge_type(e)
+		# for e in new_edges:
```

Comment:
Should this just be deleted? I noticed that there are a couple of places in pyzx where blocks of code are commented out without explanation. If these should be kept, it would be helpful to readers to have a preceding comment along the lines of "Alternate implementation of ..." or "Uncomment this block if you want to...". Otherwise, for someone new to a codebase who encounters blocks of code which are commented out, it's just puzzling why they're there.